src/Altinn-application-owner-system/Functions/Services/Implementation/QueueService.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Text;
+using System.Threading.Tasks;
+using AltinnApplicationOwnerSystem.Functions.Config;
+using AltinnApplicationOwnerSystem.Functions.Models;
+using AltinnApplicationOwnerSystem.Functions.Services.Interface;
+using Azure.Storage.Queues;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace AltinnApplicationOwnerSystem.Functions.Services.Implementation
+{
+    /// <summary>
+    /// The queue service that handles actions related to the queue storage.
+    /// </summary>
+    [ExcludeFromCodeCoverage]
+    public class QueueService : IQueueService
```

Comment:
Some services are called SI others Service, no biggie, but thought I should point it out

Reply:
Agree on consistency, but vote in favor of dropping the SI suffix.

Reply:
Agree, SI is an "Altinn2-ism" and should be dropped in Altinn 3.